cli/package.json

```diff
@@ -31,15 +31,21 @@
     "chalk": "^5.0.1",
     "cosmiconfig": "^7.0.1",
     "dotenv": "^16.0.1",
+    "lodash.get": "^4.4.2",
+    "lodash.set": "^4.3.2",
```

Comment:
This is for implementing `xata conifg get/set`. Not sure if I'll do it in this PR. I'll remove this if I don't.